velox/docs/functions/binary.rst

```diff
@@ -26,6 +26,10 @@ Binary Functions
 
     Computes the HMAC with sha512 of ``binary`` with the given ``key``.
 
+.. function:: hmac_md5(binary, key) -> varbinary
```

Comment:
The functions are added in alphabetical order in this file. Please add this function before hmac_sha1.